Ant script fails during XSLT task when called from Eclipse

When executing Ant from within Eclipse with target `-antunit-fake-pom` it fails with

```
-antunit-fake-pom:
     [xslt] Processing /Users/konradwindszus/git/policeman-tools/forbidden-apis/src/test/antunit/dummy.xml to /Users/konradwindszus/git/policeman-tools/forbidden-apis/build/test-maven/fake-pom.xml
     [xslt] Loading stylesheet pom-generator.xsl
     [xslt] : Error! Use of the extension function 'http://exslt.org/strings:split' is not allowed when the secure processing feature is set to true.
     [xslt] Failed to process /Users/konradwindszus/git/policeman-tools/forbidden-apis/src/test/antunit/dummy.xml

BUILD FAILED
/Users/konradwindszus/git/policeman-tools/forbidden-apis/build.xml:600: javax.xml.transform.TransformerException: java.lang.RuntimeException: Use of the extension function 'http://exslt.org/strings:split' is not allowed when the secure processing feature is set to true.
```

Eclipse requires to explicitly parametrize the underlying factory with

```
<factory>
     <feature name="http://www.oracle.com/xml/jaxp/properties/enableExtensionFunctions" value="true"/>
</factory>
```

to make it work.
Further information in https://bugs.eclipse.org/bugs/show_bug.cgi?id=520437.

The workaround which kicks in during Ant CLI build does not work when called from Eclipse: https://bz.apache.org/bugzilla/show_bug.cgi?id=61389

--- a/build.xml
+++ b/build.xml
@@ -598,6 +598,10 @@
     <xslt in="src/test/antunit/dummy.xml" out="build/test-maven/fake-pom.xml"
       style="src/test/antunit/pom-generator.xsl" force="true"
     >
+      <!-- necessary to make Ant build also work when triggered within Eclipse: https://bugs.eclipse.org/bugs/show_bug.cgi?id=520437-->
+      <factory>
+        <feature name="http://www.oracle.com/xml/jaxp/properties/enableExtensionFunctions" value="true"/>
+      </factory>
       <outputproperty name="indent" value="yes"/>
       <param name="jarfiles" expression="${path.main-build.xslstr}"/>
     </xslt>

